src/test/java/lotto/domain/LottosTest.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import lotto.factory.LottoFactory;
+import lotto.factory.LottosFactory;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class LottosTest {
+
+    LottosFactory factory = new LottosFactory();
+
+    @Test
+    void 로또를_구매금액만큼_만든다() {
+        // given
+        int count = 10000;
+        int money = 1000;
+
+        // when
+        Lottos lottos = factory.of(count * money, new LottoFactory());
```

Comment:
해당 메소드는 static 메소드로 선언되어있으므로 `factory` 인스턴스는 필요 없을 것 같아요.
```suggestion
        Lottos lottos = LottosFactory.of(count * money, new LottoFactory());
```
이렇게 호출하면 될 듯 합니다!